docs/content/kor/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+date: 2022-3-31
+title: 게임 사용자 인터페이스 구성 요소
+draft: false

```

Comment:
Is this file here just for the test? Are we going to commit them?

Reply:
Just the first commit is going to be merged, the second one was added so you can test the feature :D This is mentioned in the description of the PR :D